lib/dialyzer/src/dialyzer.erl

```diff
@@ -49,181 +51,189 @@
-      end,
-  doit(F).
+    F = fun() ->
+        print_plt_info(Opts)
+    end,
+    doit(F).
 
 print_plt_info(#options{init_plts = PLTs, output_file = OutputFile}) ->
-  PLTInfo = get_plt_info(PLTs),
-  do_print_plt_info(PLTInfo, OutputFile).
-
-get_plt_info([PLT|PLTs]) ->
-  String =
-    case dialyzer_plt:included_files(PLT) of
-      {ok, Files} ->
-	io_lib:format("The PLT ~ts includes the following files:\n~tp\n\n",
-		      [PLT, Files]);
-      {error, read_error} ->
-	Msg = io_lib:format("Could not read the PLT file ~tp\n\n", [PLT]),
-	throw({dialyzer_error, Msg});
-      {error, no_such_file} ->
-	Msg = io_lib:format("The PLT file ~tp does not exist\n\n", [PLT]),
-	throw({dialyzer_error, Msg})
-    end,
-  String ++ get_plt_info(PLTs);
-get_plt_info([]) -> "".
+    PLTInfo = get_plt_info(PLTs),
+    do_print_plt_info(PLTInfo, OutputFile).
+
+get_plt_info([PLT | PLTs]) ->
+    String =
+        case dialyzer_plt:included_files(PLT) of
+            {ok, Files} ->
+                io_lib:format(
+                    "The PLT ~ts includes the following files:\n~tp\n\n",
+                    [PLT, Files]
+                );
+            {error, read_error} ->
+                Msg = io_lib:format("Could not read the PLT file ~tp\n\n", [PLT]),
+                throw({dialyzer_error, Msg});
+            {error, no_such_file} ->
+                Msg = io_lib:format("The PLT file ~tp does not exist\n\n", [PLT]),
+                throw({dialyzer_error, Msg})
+        end,
+    String ++ get_plt_info(PLTs);
+get_plt_info([]) ->
+    "".
 
 do_print_plt_info(PLTInfo, OutputFile) ->
-  case OutputFile =:= none of
-    true ->
-      io:format("~ts", [PLTInfo]),
-      ?RET_NOTHING_SUSPICIOUS;
-    false ->
-      case file:open(OutputFile, [write]) of
-	{ok, FileDesc} ->
-	  io:format(FileDesc, "~ts", [PLTInfo]),
-	  ok = file:close(FileDesc),
-	  ?RET_NOTHING_SUSPICIOUS;
-	{error, Reason} ->
-	  Msg1 = io_lib:format("Could not open output file ~tp, Reason: ~p\n",
-			       [OutputFile, Reason]),
-	  throw({dialyzer_error, Msg1})
-      end
-  end.
+    case OutputFile =:= none of
+        true ->
+            io:format("~ts", [PLTInfo]),
+            ?RET_NOTHING_SUSPICIOUS;
+        false ->
+            case file:open(OutputFile, [write]) of
+                {ok, FileDesc} ->
+                    io:format(FileDesc, "~ts", [PLTInfo]),
+                    ok = file:close(FileDesc),
+                    ?RET_NOTHING_SUSPICIOUS;
+                {error, Reason} ->
+                    Msg1 = io_lib:format(
+                        "Could not open output file ~tp, Reason: ~p\n",
+                        [OutputFile, Reason]
+                    ),
+                    throw({dialyzer_error, Msg1})
+            end
+    end.
 
 cl(Opts) ->
-  F = fun() ->
-	  {Ret, _Warnings} = dialyzer_cl:start(Opts),
-	  Ret
-      end,
-  doit(F).
+    F = fun() ->
+        {Ret, _Warnings} = dialyzer_cl:start(Opts),
+        Ret
+    end,
+    doit(F).
```

Comment:
This is commonly (at least within Ericsson projects) indented with 'end' aligned under 'fun', like this:

```Erlang
cl(Opts) ->
    F = fun() ->
            {Ret, _Warnings} = dialyzer_cl:start(Opts),
            Ret
        end,
    doit(F).
```

The same with case, if, receive and try.

Reply:
Thank you for commenting and bringing this up.
`erlfmt` does not do any alignment to avoid a change of the name `F` to result in a bigger diff over multiple lines.
I hope that makes sense.